embedded_cli.h

```diff
@@ -32,6 +32,14 @@
 #define EMBEDDED_CLI_MAX_PROMPT_LEN 10
 #endif
 
+#ifndef EMBEDDED_CLI_SERIAL_XLATE
+/**
+ * Translate CR -> NL on input and output CR NL on output. This allows "natural" processing when
+ * using a serial terminal.
+ */
+#define EMBEDDED_CLI_SERIAL_XLATE 0
```

Comment:
I'd be inclined to have this enabled by default, since it seems like the more useful behaviour. I don't think there are enough users of this library to really worry about any kinds of backwards compatibility :)

Reply:
ok - I made it the default. I also added another commit which changes posix_demo.c to use raw mode (it does what cfrawmode would do). I also modified it to restore the previous settings after doing the read rather than just setting a few things.